test/testcafe/spec/Identify_spec.js

```diff
@@ -149,6 +149,7 @@ test.requestHooks(identifyMock)('should have correct display text', async t => {
 
   const signupLinkText = identityPage.getSignupLinkText();
   await t.expect(signupLinkText).eql('Sign up');
+  await t.expect(identityPage.getFooterInfo()).eql('Don\'t have an account?Sign up');
```

Comment:
Does this pass? There should be a space after the question mark.

Reply:
the space is through css is basically padding. They both are in a `span`